Solutions/Ais.Net.Specs/Ais/Net/Specs/NmeaLineToAisStreamAdapterSpecs.feature

```diff
@@ -60,7 +60,7 @@ Scenario: Three-fragment message fragments received adjacently
 	And the line to message adapter receives '\g:2-3-3451*5F\!AIVDM,3,2,9,A,<uH000167pF=2=nG0:0DRj0CQiC4jh00,0*4A'
 	# ais.kystverket.no
 	And the line to message adapter receives '\g:3-3-3451*5E\!AIVDM,3,3,9,A,0000000,0*2F'
-	Then the ais message processor should receive 1 message
+	Then INmeaAisMessageStreamProcessor.OnNext should have been called 1 times
```

Comment:
You know you can add a second attribute to the step definition (or a slightly more complex regex) to allow "1 time"  instead of "1 times"

Reply:
I even do this in other places in this project. And I even remembered this while working on this PR. And then forgot again. Gimme a sec...

Reply:
OK, fixed that.